test/features/parsoid/html2html-bodyOnly/request.json

```diff
@@ -0,0 +1,32 @@
+{
+    "bodyOnly": true,
+    "html": {
+        "headers": {
+            "content-type": "text/html;profile=mediawiki.org/specs/html/1.0.0"
+        },
+        "body": "<html>The modified HTML</html>"
+    },
+    "original": {
```

Comment:
This request looks more like a Parsoid v2 request & not like a transform request to me.